src/api/routes/address.ts

```diff
@@ -230,5 +230,22 @@ export function createAddressRouter(db: DataStore, chainId: ChainID): RouterWith
     }
   });
 
+  router.getAsync('/:stx_address/nf_id', async (req, res) => {
```

Comment:
Any thoughts on a better name for this route? It seems that `nft_events` might be more appropriate